index.js

```diff
@@ -30,7 +30,7 @@ module.exports = function conversation({name, app, appId,
   fuzzyDistance = 0.93,
   handler = (app && app.handler) || null
 }) {
-  if(handler === null) throw new Error("Must provide either an app or handler.");
+  if(handler === null) throw new Error('Must provide either an app or handler.');
```

Comment:
the if needs a pace too. you can fix this small errors by running `eslint --fix`